EasyRestProjectNetTeam2/EasyRestPages/ManageWaitersPage.cs

```diff
@@ -16,32 +22,22 @@ public ManageWaitersPage(IWebDriver driver) : base(driver)
         [FindsBy(How = How.XPath, Using = "//button[@title='Add Waiter']")]
         private IWebElement _addWaiterButton;
 
-        [FindsBy(How = How.XPath, Using = "//input[@name='name']")]
-        private IWebElement _inputName;
-
-        [FindsBy(How = How.XPath, Using = "//input[@name='email']")]
-        private IWebElement _inputEmail;
-
-        [FindsBy(How = How.XPath, Using = "//input[@name='password']")]
-        private IWebElement _inputPassword;
-
-        [FindsBy(How = How.XPath, Using = "//input[@name='phone_number']")]
-        private IWebElement _inputPhoneNumber;
-
-        [FindsBy(How = How.XPath, Using = "//span[text()='Add']/parent::button")]
-        private IWebElement _addButton;
-
-        [FindsBy(How = How.XPath, Using = "//p[text()='Name is required']")]
-        private IWebElement _inputNameValidationWarning;
+        [FindsBy(How = How.XPath, Using = " (//span[contains(@class, 'MuiTypography-root-41 MuiTypography-body1')])[7]")] // third one in the list
+        private IWebElement _nameWaiter;
 
-        [FindsBy(How = How.XPath, Using = "//p[text()='Mail is required']")]
-        private IWebElement _inputEmailValidationWarning;
+        [FindsBy(How = How.XPath, Using = "//div[contains(@class, 'MuiPaper-rounded')]/ul/li/div/span")]
+        private IList<IWebElement> _listOfWaiters;
 
-        [FindsBy(How = How.XPath, Using = "//p[text()='Password is required']")]
-        private IWebElement _inputPasswordValidationWarning;
-
-        [FindsBy(How = How.XPath, Using = "//p[text()='Phone number is required']")]
-        private IWebElement _inputPhoneNumberValidationWarning;
+        public bool CheckThatNewWaiterAppears(string NameForNewEmploee)
+        {
+            driver.Navigate().Refresh();
```

Comment:
After refreshing the page, usually, you should wait some time until the page is reloaded. Does it work stable?

Reply:
Yes, U are right. I've tried to do tests when some have app load my PC, success result was 8 from 10 times test fall down. so I try to add waiter but it was wrong decision then I added one more assert if pop up with confirmation displayed and it gives enough time for checking.